api/python/properties_prediction_d/datasets.py

```diff
@@ -0,0 +1,53 @@
+import config
```

Comment:
We should add these files to `api/python/indigo/ml`, and call it like `mpp` (molecular properties prediction).